### ISSUE TITLE
Trajectory2

### ISSUE THREAD

**Issue by itsnotklim** [itsnotklim/WPF-2021-by-Klim#2]
Показывает, что автор Гриша, но я делал программу через его компьютер, так как я не могу до сих пор разобраться с MacOS, там странно все, не дает написать файлы на Xaml, могу прикрепить скриншоты с ошибкой, но в интернете пишут, что это косяк со стороны Vs

**Comment by justjune** [comment]
На экзамене буду спрашивать масштабирование графика, запись в файлы.

**Comment by itsnotklim** [comment]
Получается, что программа не засчитана и 40 баллов по практике не набирается ?

**Comment by justjune** [comment]
Программа засчитана в том объеме, который был выполнен. См. Успеваемость. Но, 40 баллов не набирается.

**Comment by itsnotklim** [comment]
Просто мне сказали, что нужно сделать программу, которая как раз траектория - сказали, что она будет засчитана за практику

**Comment by justjune** [comment]
Кто сказал? Требования к заданиям по практике с датами исполнения лежат в onenote с начала семестра. Единственное, что можно посоветовать в этой ситуации: кто сказал --- пусть то оценку вам и поставит. :)

**Comment by itsnotklim** [comment]
Правильно-ли я понимаю, что у меня баллы порезались из-за того, что поздно сдал программу ?

**Comment by justjune** [comment]
Безусловно –– это правило, которое вам известно с первого курса.
